WordPress/src/test/java/org/wordpress/android/viewmodel/giphy/provider/TenorProviderTestUtils.kt

```diff
@@ -0,0 +1,60 @@
+package org.wordpress.android.viewmodel.giphy.provider
+
+import android.net.Uri
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import com.tenor.android.core.constant.MediaCollectionFormat
+import com.tenor.android.core.model.impl.Media
+import com.tenor.android.core.model.impl.MediaCollection
+import com.tenor.android.core.model.impl.Result
+import org.wordpress.android.viewmodel.giphy.MutableGiphyMediaViewModel
+
+class TenorProviderTestUtils {
```

Comment:
I think it would be cleaner to make this an `object TenorProviderTestUtils` and remove the companion object.